### DISCUSSION TITLE
probably feature

### DISCUSSION THREAD

**Discussion by kerzhaev** [n4ze3m/page-assist#35] [Ideas]
Would you add answer with voice from chat like "Voice control for ChatGpt". It is the best way to learn langeages offline

**Comment by n4ze3m** [comment]
Yes, we already have plans to release browser text-to-speech (TTS) support in the upcoming version. You can use (browser, OpenAI, Whisper) as the TTS provider

**Comment by n4ze3m** [comment]
Hey, web UI now has TTS support

![image](https://github.com/n4ze3m/page-assist/assets/39720973/6db75879-8ac7-416d-802b-2ffd577741c3)
